.circleci/config.yml

```diff
@@ -517,4 +511,4 @@ jobs:
       - decrypt_secrets_if_possible
       - run:
           working_directory: ~/expo/apps/test-suite
-          command: ./publish.sh
+          command: nix run expo.procps --command ./publish.sh
```

Comment:
The script starts with
```sh
#!/usr/bin/env nix-shell
#!nix-shell -I nixpkgs=../../nix --packages direnv procps git yarn -i "direnv exec . bash"
```

Doesn't this suffice, @nicknovitski?

Reply:
Reverted this change, just wanted to check if that could help. Anyway, seems like `expo publish` command run by this script cannot spawn `ps` process 🤔 

```
...
[13:07:11] Published
[13:07:11] Your URL is

https://exp.host/@exponent_ci_bot/test-suite?release-channel=1f0242223f98c392681a3cbf3e5e4597c79373d6

Error: spawn ps ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
error Command failed with exit code 1.
```

Reply:
@sjchmiela I agree, only one or the other should be necessary.  But neither are working?